javascript-carousel/main.js

```diff
@@ -0,0 +1,68 @@
+var dotRow = document.querySelector('.dot-row');
+var imgRow = document.querySelector('.img-row');
+var counter = 0;
+var images = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var changeImage = setInterval(autoChange, 3000);
+var dots = document.querySelectorAll('.dot');
+
+function autoChange() {
+  counter++;
+  if (counter >= 5) {
```

Comment:
Instead of using a magic hard-coded number here, you should make use of the `length` of the array that contains the values you want to loop over.